apps/evm/src/pages/Market/Market/OperationForm/RepayForm/index.tsx

```diff
@@ -552,8 +552,11 @@ const RepayForm: React.FC<RepayFormProps> = ({
         token: fromToken,
       });
 
+      console.log('CALLED');
+
       // Handle repay flow
       if (!isUsingSwap) {
+        console.log('no swap');
```

Comment:
A couple of logs here

Reply:
I need to pay more attention to these, you've found logs in all my latest PRs... Thanks for the catch.

Reply:
Just to ping you, they are still there! 😆

Reply:
Oh man... Now they're gone 😅